meta/featured.json

```
@@ -0,0 +1,12 @@
+{
+  "body": "This report seeks to illuminate the design process necessary to create accessible, enjoyable, and empowering data tools for place-based health equity. It also outlines basic principles of HCD, describing a general four-step process to put design into action, as well as showcasing best practices to create user-friendly designs in general.",
+  "title": "A Guide to Human-Centered Design",
+  "image": "images/human_centered_design.svg",
+  "links": [
+    {
+      "label": "Access resource →",
+      "bold": true,
+      "url": "https://www.w3docs.com/tools/code-editor/1085"
```

Use correct / updated URL for HCD PDF:
```suggestion
      "url": "https://drive.google.com/file/d/1qnXNkwat0FTbBEUZygjayCl_j7E8fIb7/view?usp=sharing"
```